packages/medusa/src/utils/admin-consts.ts

```diff
@@ -0,0 +1,3 @@
+export const ADMIN_SOURCE_DIR = "src/admin"
+export const ADMIN_RELATIVE_OUTPUT_DIR = "./public/admin"
```

Comment:
q: isn't the public folder typically associated with something else than build artefacts? maybe we can do `./dist/admin`?

Reply:
All good, let's keep it as is